dev/hal_install_spin.docker

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:14.04
+# Parameters of the GCP project Spinnaker is maintaining.
+ARG BUILD_PROJECT
+ARG BUILD_ACCOUNT
+ARG BUILD_KEY # json keyfile contents
```

Comment:
It's totally OK to assume (& document) what env vars need to be set in the container at runtime. You can pass these in either via the docker CLI, or by modifying the pod spec that's running this container in k8s.